src/Storefront/Resources/views/storefront/page/checkout/checkout-item.html.twig

```diff
@@ -112,11 +112,20 @@
                                                     </div>
                                                 {% endblock %}
 
+                                                {% set price = lineItem.price %}
+
                                                 {% block page_checkout_item_info_features %}
                                                     {% if lineItem.type == 'product' and lineItem.payload.features is not null %}
                                                         {% sw_include '@Storefront/storefront/component/product/feature/list.html.twig' with {
                                                             'features': lineItem.payload.features
                                                         } %}
+                                                    {% elseif lineItem.type == 'product' and lineItem.price.referencePrice is not null %}
+                                                        {# TODO: This branch can be removed, once the FEATURE_NEXT_6997 flag is removed #}
```

Comment:
@pweyck could we somehow mark this better, so it is recognised, when the feature flag is removed?

Reply:
Just FYI - this flag (including the `elseif` branch) has been removed already. I don't think this should still be contained in the diff after rebasing

Reply:
I just rebased it yesterday so I am a bit buffled but I guess we can solves this :D